### PR TITLE
Cleanup: move unused translation helpers to migrations

### PR DESCRIPTION
**Pull request by stefanobaghino-da**
These helpers are no longer used in the main source tree.

changelog_begin
changelog_end

### Pull Request Checklist

- [x] Read and understand the [contribution guidelines](https://github.com/digital-asset/daml/blob/master/CONTRIBUTING.md)
- [ ] Include appropriate tests
- [x] Set a descriptive title and thorough description
- [ ] Add a reference to the [issue this PR will solve](https://github.com/digital-asset/daml/issues), if appropriate
- [x] Include changelog additions in one or more commit message bodies between the `CHANGELOG_BEGIN` and `CHANGELOG_END` tags
- [ ] Normal production system change, include purpose of change in description

NOTE: CI is not automatically run on non-members pull-requests for security
reasons. The reviewer will have to comment with `/AzurePipelines run` to
trigger the build.

### CODE REVIEW

File: ledger/sandbox/src/main/scala/db/migration/postgres/V26_1__Fill_create_argument.scala [hunk 1]
```diff
@@ -5,7 +5,7 @@ package db.migration.postgres
 
 import java.io.ByteArrayInputStream
 
-import com.daml.platform.store.serialization.{ContractSerializer, ValueSerializer}
+import com.daml.platform.store.serialization.ValueSerializer
```
Comment (SamirTalwar-DA) [inline ledger/sandbox/src/main/scala/db/migration/postgres/V26_1__Fill_create_argument.scala:8]: How come you no longer have to import `ContractSerializer`?

Comment (SamirTalwar-DA) [reply ledger/sandbox/src/main/scala/db/migration/postgres/V26_1__Fill_create_argument.scala:8]: I see from the builds that you _do_. 🙃

Comment (stefanobaghino-da) [reply ledger/sandbox/src/main/scala/db/migration/postgres/V26_1__Fill_create_argument.scala:8]: Good question, looks like the compiler got there before you did. :slightly_smiling_face:

Comment (stefanobaghino-da) [reply ledger/sandbox/src/main/scala/db/migration/postgres/V26_1__Fill_create_argument.scala:8]: Addressed by https://github.com/digital-asset/daml/pull/5963/commits/3a1e438ec0990875ccdd06bd83e40728340ea4d4

**Pull request merged**